docs/notebooks/delayed-reward.ipynb

```diff
@@ -393,112 +415,266 @@
MHD6J3794AWNeVwVR1GhcXhy5dukAul0tlIiIicOHCBWRkZDxSjHxgr4W6ffs2NBqN3gUEALy9vXH+/HkzRVW9abVaTJo0CR07dkSzZs0AACkpKZDL5XB1ddUr6+3tjZSUFKlMaT8H3ToqtnnzZpw4cQLHjh0rsY71bBpXrlzBypUrMXnyZLz99ts4duwYXn/9dcjlcowYMUKqp9Lq8f569vLy0ltvY2MDd3d31vN9/vvf/0KtViM4OBjW1tbQaDT48MMPMWzYMABgXVcCU9VpSkoKAgMDS+xDt87Nze2hY2TSRDXG+PHjkZiYiIMHD5o7lMfO33//jTfeeAMxMTGws7MzdziPLa1WizZt2mDevHkAgCeffBKJiYlYtWoVRowYYeboHi9btmzBpk2b8PXXX6Np06ZISEjApEmT4Ofnx7quwdg9Z6E8PDxgbW1d4u6i1NRU+Pj4mCmq6mvChAnYvn079u7dizp16kjLfXx8UFBQgMzMTL3y99ezj49PqT8H3Toq7n5LS0tDq1atYGNjAxsbG+zfvx+ffvopbGxs4O3tzXo2AV9fXzRp0kRvWePGjZGcnAzgXj2V93fDx8cHaWlpeuuLioqQnp7Oer7P1KlT8d///heDBw9GSEgIXn75Zbz55puYP38+ANZ1ZTBVnVbm3xImTRZKLpejdevWiI2NlZZptVrExsYiLCzMjJFVL0IITJgwAT/88AN+++23Ek22rVu3hq2trV49X7hwAcnJyVI9h4WF4cyZM3q/qDExMVAqlSUuYDVV9+7dcebMGSQkJEivNm3aYNiwYdL3rOdH17FjxxJTZvz1118ICAgAAAQGBsLHx0evntVqNY4ePapXz5mZmYiPj5fK/Pbbb9BqtQgNDa2Cs6ge7t69Cysr/UuktbU1tFotANZ1ZTBVnYaFheHAgQMoLCyUysTExKBRo0aP1DUHgFMOWLLNmzcLhUIh1q9fL86dOydeffVV4erqqnd3EZVv3LhxwsXFRezbt0/cvHlTet29e1cqM3bsWFG3bl3x22+/iePHj4uwsDARFhYmrdfdCt+zZ0+RkJAgfv31V+Hp6clb4Stw/91zQrCeTeGPP/4QNjY24sMPPxQXL14UmzZtEg4ODuKrr76SykRHRwtXV1fx448/itOnT4vnnnuu1Fu2n3zySXH06FFx8OBB0bBhwxp9G3xpRowYIWrXri1NOfD9998LDw8PMW3aNKkM69p4WVlZ4uTJk+LkyZMCgFi8eLE4efKkuHbtmhDCNHWamZkpvL29xcsvvywSExPF5s2bhYODA6ccqAmWLVsm6tatK+RyuWjXrp04cuSIuUOqVgCU+lq3bp1UJjc3V/znP/8Rbm5uwsHBQTz//PPi5s2bevu5evWq6N27t7C3txceHh7irbfeEoWFhVV8NtXLg0kT69k0fv75Z9GsWTOhUChEcHCwWL16td56rVYr3n33XeHt7S0UCoXo3r27uHDhgl6ZO3fuiCFDhggnJyehVCrFqFGjRFZWVlWehsVTq9XijTfeEHXr1hV2dnaifv364p133tG7jZ11bby9e/eW+jd5xIgRQgjT1empU6dEp06dhEKhELVr1xbR0dEmiV8mxH3TmxIRERFRqTimiYiIiMgATJqIiIiIDMCkiYiIiMgATJqIiIiIDMCkiYiIiMgATJqIiIiIDMCkiYiIiMgATJqIiKrYU089hUmTJpk7DCIyEpMmInosjRw5EjKZDDKZDLa2tggMDMS0adOQl5dn7tCIqJqyMXcARESVpVevXli3bh0KCwsRHx+PESNGQCaT4aOPPjJ3aERUDbGliYgeWwqFAj4+PvD390e/fv0QHh6OmJgYAEB+fj5ef/11eHl5wc7ODp06dcKxY8ekbdevXw9XV1e9/W3btg0ymUx6P3v2bLRs2RIbN25EvXr14OLigsGDByMrK0sqk5OTg+HDh8PJyQm+vr5YtGhR5Z40EVUaJk1EVCMkJibi8OHDkMvlAIBp06bhu+++w4YNG3DixAk0aNAAERERSE9PN2q/ly9fxrZt27B9+3Zs374d+/fvR3R0tLR+6tSp2L9/P3788Ufs3r0b+/btw4kTJ0x6bkRUNZg0EdFja/v27XBycoKdnR1CQkKQlpaGqVOnIicnBytXrsTChQvRu3dvNGnSBGvWrIG9vT3Wrl1r1DG0Wi3Wr1+PZs2aoXPnznj55ZcRGxsLAMjOzsbatWvx8ccfo3v37ggJCcGGDRtQVFRUGadLRJWMY5qI6LHVrVs3rFy5Ejk5OViyZAlsbGwwYMAAnD59GoWFhejYsaNU1tbWFu3atcOff/5p1DHq1asHZ2dn6b2vry/S0tIAFLdCFRQUIDQ0VFrv7u6ORo0aPeKZEZE5MGkioseWo6MjGjRoAAD48ssv0aJFC6xduxZt27atcFsrKysIIfSWFRYWlihna2ur914mk0Gr1T5C1ERkqdg9R0Q1gpWVFd5++23MnDkTQUFBkMvlOHTokLS+sLAQx44dQ5MmTQAAnp6eyMrKQk5OjlQmISHBqGMGBQXB1tYWR48elZZlZGTgr7/+erSTISKzYNJERDXGwIEDYW1tjZUrV2LcuHGYOnUqfv31V5w7dw5jxozB3bt3ERUVBQAIDQ2Fg4MD3n77bVy+fBlff/011q9fb9TxnJycEBUVhalTp+K3335DYmIiRo4cCSsr/uklqo7YPUdENYaNjQ0mTJiABQsWICkpCVqtFi+//DKysrLQpk0b7Nq1C25ubgCKxx599dVXmDp1KtasWYPu3btj9uzZePXVV4065sKFC5GdnY0+ffrA2dkZb731FlQqVWWcHhFVMpl4sNOeiIiIiEpgGzERERGRAZg0ERERERmASRMRERGRAZg0ERERERmASRMRERGRAZg0ERERERmASRMRERGRAZg0ERERERmASRMRERGRAZg0ERERERmASRMRERGRAZg0ERERERng/wFIzOAwFqLDdgAAAABJRU5ErkJggg==",
```

Comment:
I'm not sure this plot adds much compared to the tabulated summary of results. It may be worth removing this and adding a few more rows to the table below demonstrating how much cumulative regret varies with `decay_rate`.